killbill/api/__init__.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
```

Comment:
Could you generate Copyright headers?

See https://github.com/killbill/killbill-swagger-coden/blob/HEAD/src/main/resources/killbill-java/copyright.mustache, https://github.com/killbill/killbill-swagger-coden/blob/HEAD/src/main/resources/killbill-java/api.mustache and https://github.com/killbill/killbill-swagger-coden/blob/HEAD/src/main/resources/killbill-java/model.mustache.